src/components/lobby/LobbyAwaitGame.tsx

```diff
@@ -0,0 +1,222 @@
+// tslint:disable:no-console
+import * as React from 'react';
+import styled from 'styled-components';
+import BGImage from '../../dist/images/chess-bg.jpg';
+import {CreatedGames} from '../../store/MockData';
+
+const ServerData = CreatedGames;
+const mockGame = {creator : "Anonymous", creatorColor: "",  opponent : "Anonymous", gameId : 2, gameType : "Classic", gameTime : 5, gameAddTime : 15, gameStarted : false, player1Ready: false, player2Ready: false,}; 
+
+
+interface IStateType  {
+  Player1Ready: boolean;
+  Player2Ready: boolean;
+  IsPlayer1: boolean;
+  gameId: number;
+ 
+}
+class LobbyAwaitGameComponent extends React.Component<any, IStateType> {
+
+  constructor (props: any) {
+    super(props)
+    this.state = {             
+      gameId: this.props.match.params.id,     
+      IsPlayer1: true,
+      Player1Ready: false,
+      Player2Ready: false,
+    };
+    
+}
+
+public SetReadyPlayer1 = (GameData : any) => {
+    this.setState({Player1Ready: true});
+}
+
+public SetReadyPlayer2 = (GameData : any) => {
+    this.setState({Player2Ready: true});
+}
+
+public MakeReady = () => {
+   if(this.state.IsPlayer1) {
+       console.log('Change player 1 to ready in DB');
+   }else {
+       console.log('Change player 2 to ready in DB');
+   }
+}
+
+public FindMatchById = () => {
+    const findGame = ServerData.find(MatchId => MatchId.gameId === +this.state.gameId);
+
+    if(findGame) {
+        return findGame;
+    }else {
+      return mockGame
+    }
+}
+
+public render() {  
+
+    const gameData = this.FindMatchById();
+    const Player1ReadyColor = this.state.Player1Ready ? "green" : "red";
+    const Player2ReadyColor = this.state.Player2Ready ? "green" : "red";
+    const player1Text = this.state.Player1Ready ? "Ready" : "Not Ready";
+    const player2Text = this.state.Player2Ready ? "Ready" : "Not Ready";
+    const readyBtnColor = this.state.IsPlayer1 ? Player1ReadyColor : Player2ReadyColor;
+    const readyBtnText = this.state.IsPlayer1 ? player1Text : player2Text;   
+    
+    if (gameData.player1Ready === true && this.state.Player1Ready !== true) {        
+         this.SetReadyPlayer1(gameData);
+     };
+
+    if (gameData.player2Ready === true && this.state.Player2Ready !== true) {        
+        this.SetReadyPlayer2(gameData);
+     };
+
+    return (
+        
+        <Container>
+            <Content>
+                 <Header>
+                  Waiting for players
+                  </Header>
+                <Players>
+                   <Player>
+                       <PlayerName>{gameData.creator}</PlayerName>
+                       <PlayerReadyButton style={{backgroundColor: Player1ReadyColor,}}>{player1Text}</PlayerReadyButton>
```

Comment:
The PlayerReadyButton component can be refactorized and have props to pass for the style color. Check with Malin how to do that with Styled Components